config/areas/settings/dialogs.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+use Kirby\Cms\Find;
+use Kirby\Panel\Field;
+
+$languageDialogFields = [
+    'name' => [
+        'label'    => t('language.name'),
+        'type'     => 'text',
+        'required' => true,
+        'icon'     => 'title'
+    ],
+    'code' => [
+        'label'    => t('language.code'),
+        'type'     => 'text',
+        'required' => true,
+        'counter'  => false,
+        'icon'     => 'globe',
+        'width'    => '1/2'
+    ],
+    'direction' => [
+        'label'    => t('language.direction'),
+        'type'     => 'select',
+        'required' => true,
+        'empty'    => false,
+        'options'  => [
+            ['value' => 'ltr', 'text' => t('language.direction.ltr')],
+            ['value' => 'rtl', 'text' => t('language.direction.rtl')]
+        ],
+        'width'    => '1/2'
+    ],
+    'locale' => [
+        'label' => t('language.locale'),
+        'type'  => 'text',
+    ],
+];
+
+return [
+
+    // create language
+    'languages/create' => [
+        'load' => function () use ($languageDialogFields) {
+            return [
+                'component' => 'k-language-dialog',
+                'props' => [
+                    'fields' => $languageDialogFields,
+                    'submitButton' => t('language.create'),
+                    'value' => [
+                        'name'      => '',
+                        'code'      => '',
+                        'direction' => 'ltr',
+                        'locale'    => ''
+                    ]
+                ]
+            ];
+        },
+        'submit' => function () {
+            kirby()->languages()->create(get());
```

Comment:
I feel like the props should be allowlisted to avoid weird prop injections:

```suggestion
        'submit' => function () use ($languageDialogFields) {
            $props = array_intersect_key(get(), $languageDialogFields);
            kirby()->languages()->create($props);
```